.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
What is the purpose of this file?

Reply:
It was there in my branch when I created it. So I thought it's an inbuilt file.

Reply:
No such file should be committed to git as these are intellij specific files. Please add .gitignore to ignore any files in .idea folder. If its created in git please delete it

Reply:
Okay